app/controllers/admin/users_controller.rb

```diff
@@ -170,6 +171,16 @@ def remove_org_credits
       Credit.remove_from(org, amount)
     end
 
+    def set_related_reactions
+      user_article_ids = @user.articles.ids
+      user_comment_ids = @user.comments.ids
+      @related_vomit_reactions = Reaction.where(reactable_type: "Comment", reactable_id: user_comment_ids,
+                                                category: "vomit")
+        .or(Reaction.where(reactable_type: "Article", reactable_id: user_article_ids, category: "vomit"))
+        .includes(:reactable)
+        .order(created_at: :desc).limit(15)
```

Comment:
Does `limit(15)` mean we are limiting the results to the first 15? If so, why? Or another way to ask is, could Forem admins have access to records not contained in this 15?

Reply:
Yes, the `limit(15)` limits the records to 15 records per user. I have done this for scalability reasons and to follow the convention of the other sections similar to this in `/admin/users` like the "Recent Emails" and "Recent Notes" section, but I'm open to suggestions!

Reply:
I think limiting to 15 is fine -- we might want to paginate it in a future PR.